packages/smooth_app/lib/pages/preferences_v2/app_bars/logged_in_app_bar.dart

```diff
@@ -0,0 +1,237 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:openfoodfacts/openfoodfacts.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/pages/preferences_v2/app_bars/app_bar_background.dart';
+import 'package:smooth_app/pages/preferences_v2/app_bars/search_bottom_bar.dart';
+import 'package:smooth_app/themes/smooth_theme.dart';
+import 'package:smooth_app/themes/smooth_theme_colors.dart';
+
+const double TOOLBAR_HEIGHT = 92.0;
+
+class LoggedInAppBar extends StatelessWidget {
+  const LoggedInAppBar();
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    final SmoothColorsThemeExtension themeExtension =
+        context.extension<SmoothColorsThemeExtension>();
+    final AppLocalizations appLocalizations = AppLocalizations.of(context);
+
+    return SliverAppBar(
+      title: Row(
+        children: <Widget>[
+          Container(
+            width: 68.0,
+            height: 68.0,
+            padding: const EdgeInsetsDirectional.all(8.0),
+            decoration: const BoxDecoration(
+              shape: BoxShape.circle,
+              color: Colors.white,
+            ),
+            child: Center(
+              child: SvgPicture.asset(
+                'assets/app/release_icon_light_transparent_no_border.svg',
+              ),
+            ),
+          ),
+          const SizedBox(width: 12.0),
+          Expanded(
+            child: Column(
+              children: <Widget>[
+                Row(
+                  children: <Widget>[
+                    Flexible(
+                      child: Text(
+                        OpenFoodAPIConfiguration.globalUser?.userId ?? '',
```

Comment:
Is `null` an option?
May make sense to put the `userId` as a `String` parameter of the Widget.